src/river/river.py

```
@@ -0,0 +1,337 @@
+
+    transformed_points = [tr.transform(point) for point in points]
+
+    # Находим новые границы
+    x_coords = [p.x() for p in transformed_points]
+    y_coords = [p.y() for p in transformed_points]
+
+    return f"{min(x_coords)}, {max(x_coords)}, {min(y_coords)}, {max(y_coords)}"
+
+
+def river(project_folder):
+    set_project_crs()
```

Ruff (0.8.2)

`set_project_crs` may be undefined, or defined from star imports

(F405)